cubi_tk/isa_tpl/__init__.py

```diff
@@ -176,8 +176,8 @@ def run_cookiecutter(tpl, args, _parser=None, _subparser=None, no_input=False):
     logger.info(args)
 
     output_dir = os.path.realpath(args.output_dir)
-    output_base = os.path.dirname(args.output_dir)
-    extra_context["i_dir_name"] = os.path.basename(output_dir)
+    output_base = os.path.dirname(output_dir)
+    extra_context["__output_dir"] = os.path.basename(output_dir)
```

Comment:
I think the change to `output_base` (now affected by realpath) could potentially break things if one invokes the command in wierd way with symlinks or a target in a completely different / nested destination?

Reply:
I don't see how. Tried several edge cases and it seems to work. Prior to this change `output_base` was not derived from a normalised path which is why trailing slashes did change its value.

Reply:
I think that you can construct instances where the change to `output_base` affects the destination of the output files.
For example, in the following directory structure:
```
a
├── b
│   └── dangling -> no_path
└── b2
    └── dangling -> ../../a_new/b/dangling
```
where `dangling` is a symlink which target is missing, then when the `output_dir` argument is set to `a/b2/dangling`, the old code will create the output files in the directory `a/b2/no_path`, while the new code will create them under `a/b/no_path`.

In that situation, and if the folder `a/b2/no_path` already exists, the old code throws an error, but not the new one, and the user may overlook the newly generated files.

However, this is pretty much the only situation I could imagine where the two versions of the code would lead to different results. And this situation is also fairly screwed up in my opinion, and I am not sure that the old way of generating files is much better.